src/umd/ReactDOMUMDEntry.js

```diff
@@ -11,24 +11,24 @@
 
 'use strict';
 
+var React = require('React');
```

Comment:
Ah yea, that get's rewritten to the ReactUMDShim. Sorry, just experiencing some light PTSD thinking about making AMD again. Carry on then :)